includes/class-wc-calypso-bridge-addons-screen.php

```diff
@@ -103,4 +107,44 @@ public static function output() {
 		 */
 		include_once WC_ABSPATH . 'includes/admin/views/html-admin-page-addons.php';
 	}
+
+	/**
+	 * Install Woo Tax from Extensions screens.
+	 *
+	 * @since x.x.x
+	 */
+	public static function install_woo_tax_addon() {
+		check_admin_referer( 'install-addon_woocommerce-tax' );
+
+		$services_plugin_id = 'woocommerce-tax';
+		$services_plugin    = array(
+			'name'      => __( 'Woo Tax', 'woocommerce' ),
+			'repo-slug' => 'woocommerce-tax',
+		);
+
+		WC_Install::background_installer( $services_plugin_id, $services_plugin );
```

Comment:
I think you're already aware, but just precaution that background_installer only supports [wp.org listed plugins](https://github.com/woocommerce/woocommerce/blob/7dc8dd63bad9069307b095c05798a6eea6f40d1d/plugins/woocommerce/includes/class-wc-install.php#L2180-L2181), in case we're releasing this prior making the plugin public.